__sdk__.js

```diff
@@ -94,6 +94,8 @@ module.exports = {
     entry: "./src/shopper-insights/interface",
   },
   "three-domain-secure": {
+    globals,
+    automatic: true,
```

Comment:
We don't need to `automatic: true` anymore, correct? I believe we were testing this when we were running into the zoid xprops issue.

Reply:
`automatic: true` will return this component for everyone that loads the JS SDK. The exported [setup()](https://github.com/paypal/paypal-checkout-components/pull/2452/files#diff-c310050612136b5f5121d74da169666e0bf8d687f51c7275de09e3a4bafef6deR24) method is what will automatically create the zoid component when the component is loaded. 

This will probably be fine for now since this component is still protected but we should remove this before we allow this in Production. If we don't, `paypal.ThreeDomainSecureClient` will be available to everyone even if they don't pass `three-domain-secure` into the components query param.

Reply:
we should probably remove it now because even though its hidden, it will show up in all bundles served. I'm sure its super small but might as well not ship javascript if we don't have to

Reply:
That makes sense. I've removed that flag. Thanks both for clarifying :)